tooling/nargo_cli/tests/compile_failure/dup_trait_implementation/src/main.nr

```diff
@@ -1,28 +1,26 @@
 use dep::std;
 
 trait Default {
-    fn default(x: Field, y: Field) -> Self;
+    fn default(x: Field, y: Field) -> Field;
 }
 
 struct Foo {
     bar: Field,
     array: [Field; 2],
 }
 
-// Duplicate trait implementations should not compile
 impl Default for Foo {
-    fn default(x: Field,y: Field) -> Self {
-        Self { bar: x, array: [x,y] }
+    // Duplicate trait methods should not compile
+    fn default(x: Field, y: Field) -> Field {
+        y + 2 * x
     }
-}
-
-// Duplicate trait implementations should not compile
-impl Default for Foo {
-    fn default(x: Field, y: Field) -> Self {
-        Self { bar: y, array: [y,x] }
+    // Duplicate trait methods should not compile
+    fn default(x: Field, y: Field) -> Field {
+        y + 2 * x
```

Comment:
Why was this test changed to a different test?

Reply:
Sorry, this should've been in the previous PR, we've made a mistake when we split it in two.

It tests the same thing, but it's been simplified to use less language features, to reduce the chance of a false positive, since `compile_failure` tests result in a passing test on any compiler error.